deploy/manifests/scc.yaml

```diff
@@ -0,0 +1,33 @@
+# Create SCC for Gateway resources
```

Comment:
per another comment by Kate, this SCC should live in the helm chart, not be a separate manifest (though we can also generate the manifest (with rbac) for users who install using manifests). That way if I'm a helm user, I can just `helm install` and set `platform=openshift`, and the SCC is automatically installed.